sys/net/gnrc/network_layer/sixlowpan/iphc/gnrc_sixlowpan_iphc.c

```diff
@@ -737,6 +737,12 @@ void gnrc_sixlowpan_iphc_recv(gnrc_pktsnip_t *sixlo, void *rbuf_ptr,
     if (rbuf != NULL) {
         ipv6 = rbuf->pkt;
         assert(ipv6 != NULL);
+        if ((ipv6->size < sizeof(ipv6_hdr_t)) &&
```

Comment:
Shouldn't this be

```suggestion
        if ((ipv6->size < sizeof(ipv6_hdr_t)) ||
```

Reply:
ah no this tries to embiggen the buffer if it's too small to fit an IPv6 header, right?

Reply:
Yepp!